### ISSUE TITLE
How do I install/build/run the OCI Extirpater?

### ISSUE THREAD

**Issue by lucasjellema** [therealcmj/ociextirpater#7]
Dear reader, 
I found this tool through the blog article. I would like to use it. However, I do not know how to do so. Can I run it in Cloud Shell? (one of the issues suggests that that is not possible). Then what would be a suitable environment? 

Is there a command line command to run the tool and feed it with a compartment OCID?
I was looking for instructions in the article, the Readme or some other place. Can you make a suggestion?

thanks.
Lucas

**Comment by wanax1** [comment]
Hi lucasjellema,

You could execute the script ociextirpate.py and get the following message.
ociextirpate.py [-h] [-cf CONFIG_FILE] [-cp CONFIG_PROFILE] [-ip] [-dt] [-log LOG_FILE] [-force] [-debug] [-skip_delete_compartment] [-rg REGIONS] -c COMPARTMENT [-o OBJECTS] [-t THREADS]

**Comment by lucasjellema** [comment]
Thanks for that instruction. I tried to run the ociextirpater and ran into an issue that I cannot explain
Can you help me?

```
2024-05-24 12:20:35,342 MainThread    INFO config:__init__ -> Home region: us-ashburn-1
2024-05-24 12:20:35,343 MainThread    INFO config:__init__ -> 7 Regions to be extirpated: ['eu-amsterdam-1', 'me-dubai-1', 'eu-frankfurt-1', 'us-ashburn-1', 'ap-tokyo-1', 'ap-singapore-1', 'us-sanjose-1']
2024-05-24 12:20:35,343 MainThread    INFO ociextirpate:<module> -> Configured
2024-05-24 12:20:35,343 MainThread    INFO ociextirpate:<module> -> 36 clients
2024-05-24 12:20:35,560 MainThread    INFO OCIClient:__init__ -> Initializing clients...
Traceback (most recent call last):
  File "ociextirpate.py", line 35, in <module>
    o = cls( cfg )
  File "/home/lucas/ociextirpater/ociextirpater/OCIClient.py", line 37, in __init__
    self._init_regional_client(config.ociconfig, config.home_region)
  File "/home/lucas/ociextirpater/ociextirpater/OCIClient.py", line 18, in _init_regional_client
    logging.debug("Old region in this OCI config dict: {}".format(rconfig["region"]))
KeyError: 'region'
```

![image](https://github.com/therealcmj/ociextirpater/assets/1296324/6f786184-a95f-49e2-915e-4866ea26b02b)

**Comment by wanax1** [comment]
Hi lucasjellema,
Yes, at present the script can't work in Cloud Shell. Christopher might will fix it soon.
Please check the https://github.com/therealcmj/ociextirpater/issues/3.

**Comment by therealcmj** [comment]
Fixed in bffaa27

add -dt to your command line arguments for now to tell it "i'm running in cloud shell" and it should work!
